src/views/TabView/TabView.js

```diff
@@ -185,6 +184,9 @@ class TabView extends React.PureComponent<Props> {
 
     const tabBarVisible =
       options.tabBarVisible == null ? true : options.tabBarVisible;
+      
+    const swipeEnabled =
+      options.swipeEnabled == null ? this.props.swipeEnabled : options.swipeEnabled; 
```

Comment:
`swipeEnabled` should have a default prop set to `true` so that this wouldn't be a breaking change.

Reply:
The value of ```props.swipeEnabled``` is taken from the TabNavigator.js, where it is already set to Presets value (for Android is ```true```, but for iOS is ```false```)

Reply:
Ah okay great! I had confused myself with the `tabBarVisible` above. Once the docs are added I will merge. Thank you!

Reply:
Adds a new option in the ```Screen Navigation Options``` in ```TabNavigator```:

**```swipeEnabled ```**
True or false to enable or disable swiping between tabs, if not set then defaults to TabNavigatorConfig option swipeEnabled .